rtpsender.go

```diff
@@ -201,3 +204,48 @@ func (r *RTPSender) hasSent() bool {
 		return false
 	}
 }
+
+// collectStats collect rtpsender stats for this instance
+func (r *RTPSender) collectStats(report *statsReportCollector) {
+
+	if r.Track() == nil {
+		return
+	}
+
+	report.Collecting()
+
+	switch r.Track().Kind() {
+	case RTPCodecTypeAudio:
+		r.mu.RLock()
+		totalSamplesSent := r.track.totalSamplesSent
+		totalSamplesDuration := r.track.totalSamplesDuration
+		r.mu.RUnlock()
+		stats := AudioSenderStats{
+			Timestamp:            statsTimestampFrom(time.Now()),
+			Type:                 StatsTypeSender,
+			ID:                   r.statsID,
+			TrackIdentifier:      r.track.ID(),
+			Kind:                 r.track.Kind().String(),
+			AudioLevel:           0,     // Todo Audio level
+			TotalAudioEnergy:     0,     // Todo Total Audio Energy
+			VoiceActivityFlag:    false, // Todo Total voice activity flag
+			TotalSamplesDuration: totalSamplesDuration,
+			TotalSamplesSent:     uint64(totalSamplesSent),
+		}
+
+		report.Collect(stats.ID, stats)
+	case RTPCodecTypeVideo:
+		stats := VideoSenderStats{
+			Timestamp:      statsTimestampFrom(time.Now()),
+			Type:           StatsTypeSender,
+			ID:             r.statsID,
+			FramesCaptured: 0, // Todo Frames Captured
```

Comment:
Can you do `// TODO ...` for consistency. also, lets make sure to track that these are missing in the